package.json

```diff
@@ -35,20 +40,23 @@
     "url": "https://github.com/bufferoverflow/verdaccio-gitlab/issues"
   },
   "dependencies": {
-    "node-gitlab-api": "^2.2.0",
+    "gitlab": "^3.2.2",
```

Comment:
Usually, I do not use `caret`, just matter of taste. I think is much better have the full control.

Reply:
yep, I fix the versions.